commit 5fd26bf7da4f5c5aa97e86ef2dc4f4c191dc6fdb (refs/changes/10/3610/3)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-03-19T15:39:54-04:00 (1 year, 7 months ago)
    
    TRITON-240 zoneevent should be called with an identifier string
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/lib/vm.js b/lib/vm.js
index 5715c40..2f7bbf7 100644
--- a/lib/vm.js
+++ b/lib/vm.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -204,7 +204,7 @@ EventWatcher.prototype.startZoneWatcher = function () {
 
     var watcher = this.zoneeventWatcher = spawn(
         '/usr/vm/sbin/zoneevent',
-        [],
+        ['-i', 'net-agent'],
         {'customFds': [-1, -1, -1]});
 
     log.info('zoneevent running with pid ' + watcher.pid);
